Assets/MixedRealityToolkit/_Core/Devices/Simulator/SimulatedDevice.cs

```diff
@@ -16,7 +16,7 @@
 namespace Microsoft.MixedReality.Toolkit.Internal.Devices.Simulator
 {
     // TODO - Implement
-    public class SimulatedDevice : IMixedRealityDevice
+    class SimulatedDevice : IMixedRealityDevice
```

Comment:
`internal`? (There are some other cases of this too, but I'll leave one comment for discussion.)

Reply:
this seems reasonable, mostly to avoid too much churn if we determine that the input system assembly needs to access these classes.